### PR TITLE
fix IE again

### PR DESCRIPTION
**Pull request by pandafulmanda**
axios was assuming native Promises :(.  switching to jquery anyway since cnx uses jquery

### CODE REVIEW

**Comment by philschatz** [comment]
I still see promise-like-things in there... :smile:

File: src/api/loader.coffee [hunk 1]
```diff
@@ -13,15 +13,19 @@ getAjaxSettingsByEnv = (isLocal, baseUrl, setting, eventData) ->
 
   {data, change} = eventData
   apiSetting = _.pick(setting, 'url', 'method')
+  apiSetting.dataType = 'json'
+  apiSetting.contentType = 'application/json;charset=UTF-8'
+
   if _.includes(METHODS_WITH_DATA, apiSetting.method)
-    apiSetting.data = change or data
+    apiSetting.data = JSON.stringify(change or data)
```
Comment (nathanstitt) [inline src/api/loader.coffee:21]: jQuery would handle the serialization of an object as json internally, but doesn't hurt to be explicit.

**Comment by nathanstitt** [comment]
:+1: works fine locally for me

File: src/api/loader.coffee (hunk 1 above)
Comment (pandafulmanda) [inline src/api/loader.coffee:21]: it wasnt :(

**Pull request merged**